requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --extra=dev --extra=jupyter --output-file=requirements.txt pyproject.toml
+#    pip-compile
```

Comment:
Here I ran `pip-compile --upgrade` and it dropped all of the optional dependencies :/